lib/dor/workflow/client/requestor.rb

```diff
@@ -18,13 +18,15 @@ def initialize(connection:)
         # @param [String] payload body for (e.g. put) request
         # @param [Hash] opts addtional headers options
         # @return [Object] response from method
+        # @raise [Dor::WorkflowException,Dor::MissingWorkflow] if there are Faraday exceptions
         def request(uri_string, meth = 'get', payload = '', opts = {})
           response = send_workflow_resource_request(uri_string, meth, payload, opts)
           response.body
         rescue Faraday::Error => e
+          status = e.try(:response)&.fetch(:status, nil)
```

Comment:
Returns `nil` unless `e` responds to `:response` and returns a hash with a key of `:status`.

Reply:
could you add a comment here?  line 26 is a little magic looking to me.

Reply:
Done! Pushed.